erdpy/config.py

```diff
@@ -177,6 +177,9 @@ def get_defaults() -> Dict[str, Any]:
         "dependencies.mcl_signer.urlTemplate.osx": "https://github.com/ElrondNetwork/elrond-sdk-go-tools/releases/download/{TAG}/mcl_signer_{TAG}_macos-latest.tar.gz",
         "dependencies.wasm-opt.tag": "latest",
         "dependencies.twiggy.tag": "latest",
+        "dependencies.wabt.tag": "1.0.27",
+        "dependencies.wabt.urlTemplate.linux": "https://github.com/WebAssembly/wabt/releases/download/1.0.27/wabt-1.0.27-ubuntu.tar.gz",
+        "dependencies.wabt.urlTemplate.osx": "https://github.com/WebAssembly/wabt/releases/download/1.0.27/wabt-1.0.27-macos.tar.gz",
```

Comment:
Could we use `latest` here? Can Github version querying be reused for this?

Reply:
Here we download an _asset_ of a _release_, not the source code under a tag, and this does not work well with the current implementation of `StandaloneModule.get_latest_release()` (e.g. `v`-prefix-related issues). For the moment, we could keep a fixed version here.